src/components/common/MaterialIcon/MaterialIcon.scss

```diff
@@ -38,11 +43,10 @@
   &.md-48, &.extralarge { font-size: 2em; }
 }
 
-
 /* Rules for using icons as black on a light background. */
 .base.md-dark { color: rgba(0, 0, 0, 0.54); }
 .base.md-dark.md-inactive { color: rgba(0, 0, 0, 0.26); }
 
 /* Rules for using icons as white on a dark background. */
 .base.md-light { color: rgba(255, 255, 255, 1); }
-.base.md-light.md-inactive { color: rgba(255, 255, 255, 0.3); }
+.base.md-light.md-inactive { color: rgba(255, 255, 255, 0.3); }
```

Comment:
MIssing new line

Reply:
Probably also something that can be included in the linter
https://eslint.org/docs/latest/rules/eol-last